libminifi/src/Site2SiteClientProtocol.cpp

```diff
@@ -1240,6 +1257,88 @@ void Site2SiteClientProtocol::transferFlowFiles(
   return;
 }
 
+void Site2SiteClientProtocol::transferBytes(core::ProcessContext *context, core::ProcessSession *session, uint8_t *payload, int length,
```

Comment:
how can i do a std::move from a string to a std::vector<uint8_t> without copy?
 http://stackoverflow.com/questions/10445042/stdmove-between-stdstring-and-stdvectorunsigned-char
pass the raw byte looks OK to me and make the code much simple in this case instead of convert from one type to another.